### PR TITLE
Added Timer and Drink components

### PR DESCRIPTION
**Pull request by alafiand**
Adds `react-countdown` package, two components, and alters `CurrentSongView.tsx` to provide rough sequence of song loads, action, and moving on to next song.

### CODE REVIEW

File: frontend/src/components/CurrentSongView.tsx [hunk 1]
```diff
@@ -29,7 +39,7 @@ function CurrentSongView() {
     !asyncSong.loading &&
     !asyncSong.error &&
     asyncSong.result?.data?.content
-      .replaceAll("[tab]", "")
+      .replaceAll("[*tab]", "")
```
Comment (samdatkins) [inline frontend/src/components/CurrentSongView.tsx:42]: what's this for, did something change with the way these are coming over the wire?

File: frontend/src/components/Timer.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import Countdown from "react-countdown";
+import { Text, keyframes } from "@chakra-ui/react";
+import { useState } from "react";
+
+interface TimerThings {
```
Comment (samdatkins) [inline frontend/src/components/Timer.tsx:5]: Same here if you choose to rename the other type

File: frontend/src/components/Action.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Text, keyframes } from "@chakra-ui/react";
+
+interface ActionThings {
```
[the remark this answers is not in the document]
Comment (alafiand) [reply frontend/src/components/Action.tsx:3]: Good to know, updated!

File: frontend/src/components/Action.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  actionText: string;
+}
+
+export default function Action({ actionText }: ActionThings) {
```
[the remark this answers is not in the document]
Comment (alafiand) [reply frontend/src/components/Action.tsx:7]: Agreed! Updated to `ActionPrompt`

File: frontend/src/components/Timer.tsx (hunk 2 above)
Comment (alafiand) [reply frontend/src/components/Timer.tsx:5]: Addressed!

File: frontend/src/components/CurrentSongView.tsx (hunk 1 above)
Comment (alafiand) [reply frontend/src/components/CurrentSongView.tsx:42]: This was a mistake, I thought we might be able get away with a single `replaceAll` by using a wildcard, it didn't work and I missed it in my review.

**Pull request merged**